recipes/mitos/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set name = "mitos" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: de00eecb47f5a67c4a27104c87b4d6e26f1c9eddc86369864f7aed1b486c21c6
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  noarch: python
```

Comment:
I think this needs to be `noarch: generic` as you have also non-python deps afaik.

Reply:
changed